es-vue-base/src/lib-validators/index.js

```diff
@@ -34,3 +34,90 @@ export function vuelidateEmail(emailAddress) {
 export function vuelidatePhone(number) {
     return !helpers.req(number) || /^[+]?[(]?[0-9]{3}[)]?[-\s.]?[0-9]{3}[-\s.]?[0-9]{4,6}$/.test(number);
 }
+
+/**
+ * @param { string } param
+ * @returns { boolean } if string contains a number
+ */
+export const hasNumber = (param) => /[0-9]/.test(param);
```

Comment:
I wonder if we should support the amount of "things" required in these validators. This will only tell us if a number is present but lets say we want to check if at least 2 numbers are present.

Reply:
![Screen Shot 2022-11-04 at 3 41 10 PM](https://user-images.githubusercontent.com/114260/200984617-a646c503-cb62-4d38-b026-e7e953bebfc8.png)

So this was based upon the password validation from cognito. Their interface seems to allow checking boxes to require the various "things", but there doesn't seem to be hooks for *number of things*.

Is the suggestion to tweak this to be something similar to `minLength` i.e.

```js
hasNumbers = (x=1) => (param) => [...param.matchAll(/[0-9]/g)].length >= x
```